Dockerfile.dapper

```diff
@@ -32,7 +32,7 @@ ENV GOLANG_ARCH_amd64=amd64 GOLANG_ARCH_arm=armv6l GOLANG_ARCH=GOLANG_ARCH_${ARC
 # ginkgo         | tests
 # goimports      | code formatting
 RUN apt-get -q update && \
-    apt-get install -y gcc git curl docker.io && \
+    apt-get install -y gcc git curl docker.io mercurial make && \
```

Comment:
Mercurial seems to take a really long time to install FYI. My build hangs on it, with one thread at 100%, for what feels like minutes.